launch/dynamo-run/src/subprocess/vllm_inc.py

```diff
@@ -185,7 +186,11 @@ def cmd_line_args():
     args = parser.parse_args()
 
     config = Config()
-    config.model = args.model
+    config.model_path = args.model_path
+    if args.model_name:
+        config.model_name = args.model_name
+    else:
+        config.model_name = None
```

Comment:
Is the key here to be `None` instead of empty string `""` when being interpreted as an `Optional` on Rust/pyo3 side?

Reply:
or maybe default `= None` in `Config`or in the argparse default may just let you remove the `else` block here

Reply:
I am hoping that moving these to Python will allow other people to contribute to the engines, so I have to leave some low hanging fruit :-)